website/cypress/e2e/tasks/random.cy.ts

```diff
@@ -44,44 +44,11 @@ describe("handles random tasks", () => {
                 break;
               }
               case "label-task": {
-                cy.get('[data-cy="label-group-item"]')
-                  .first()
-                  .invoke("attr", "data-label-type")
-                  .then((label_type) => {
-                    const parent = cy
-                      .get('[data-cy="label-group-item"]')
-                      .first();
-                    cy.log("Label type", label_type);
-
-                    switch (label_type) {
-                      case "slider": {
-                        // Clicking on the slider will set the value to about the middle where it clicks
-                        parent
-                          .get('[aria-roledescription="slider"]')
-                          .first()
-                          .click();
-
-                        cy.get('[data-cy="review"]').click();
-
-                        cy.get('[data-cy="submit"]').click();
-
-                        break;
-                      }
-                      case "radio": {
-                        // Clicking on the slider will set the value to about the middle where it clicks
-                        parent
-                          .get('[aria-roledescription="radio-button"]')
-                          .last()
-                          .click();
+                // TODO: Reimplement to click on a aria-roledescription="radio" in each incomplete aria-roledescription="radiogroup"
+                cy.get('[aria-roledescription="radiogroup"]');
```

Comment:
we can assign custom data properties for testing only, similar to what we had previously `data-cy="label-group-item"`

Reply:
> 

Rather than radio buttons or new radio group buttons, we could use a Chakra [Slider](https://chakra-ui.com/docs/components/slider#discrete-sliders) with discrete options and [custom tics](https://chakra-ui.com/docs/components/slider#slider-with-custom-labels-and-marks)

Reply:
@fozziethebeat, would this still look similar or would it look like a slider again. I couldn't see a way that the custom ticks reproduce the same N options interface, Chakra's examples look like they don't change how the actual sliders work or appear.

Reply:
I'm not sure it's a great option but you can customize the slider appearance and option.  It still ultimately would be a slider however.

Reply:
> we can assign custom data properties for testing only, similar to what we had previously `data-cy="label-group-item"`

I've made an attempt at this, but it is a little odd as you can't use data-cy on components that don't render to the dom and expect cypress to find them so I had to pass it down. For finding the options in the radio group I'm still using the aria role as that should work regardless of how we implement the actual radio buttons.